libcxx/include/tuple

```diff
@@ -1153,9 +1154,16 @@ struct __tuple_equal<0> {
 };
 
 template <class... _Tp, class... _Up>
+#    if _LIBCPP_STD_VER >= 26
+  requires(requires(const _Tp& __t, const _Up& __u) {
+            { __t == __u } -> __boolean_testable;
+          } && ...) && (sizeof...(_Tp) == sizeof...(_Up))
```

Comment:
I think we should use `__all` here. Otherwise we're potentially breaking currently working code, since fold expressions have a significantly lower limit than the number of allowed template arguments.

Reply:
It does, but it's not yet available in most compilers. Note that I'm only asking for `tuple` to be changed, since it's notorious for being instantiated with a huge number of arguments.

Reply:
OK. In that case. I'll split tuple in a separate patch.